dockstore-common/generated/src/main/resources/pom.xml

```diff
@@ -19,7 +19,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>io.dockstore</groupId>
   <artifactId>dockstore-common</artifactId>
-  <version>1.11.6-SNAPSHOT</version>
+  <version>1.11.7-SNAPSHOT</version>
```

Comment:
anyone know why this is not matching with the rest?

Reply:
We probably need to update the documentation. 
The release(r?) needs to run `mvn clean install` once more after doing the release to regenerate the generated files that snyk relies on.